modules/sfp_spyse.py

```diff
@@ -430,13 +440,144 @@ def handleEvent(self, event):
+
+                    domain_whois = domain_item.get("whois_parsed")
+                    if domain_whois:
+                        domain_whois_registrar = domain_whois.get("registrar")
+                        if domain_whois_registrar:
+                            if domain_whois_registrar.get("registrar_name"):
+                                evt = SpiderFootEvent('DOMAIN_REGISTRAR',
+                                                      domain_whois_registrar["registrar_name"], self.__name__,
+                                                      event)
+                                self.notifyListeners(evt)
+
+                    domain_http_extract = domain_item.get("http_extract")
+                    if domain_http_extract:
+
+                        if domain_http_extract.get("http_status_code"):
+                            evt = SpiderFootEvent('HTTP_CODE', str(domain_http_extract["http_status_code"]),
+                                                  self.__name__, event)
+                            self.notifyListeners(evt)
+
+                        domain_emails = domain_http_extract.get("emails")
+                        if domain_emails:
+                            for email in domain_emails:
+                                evt = SpiderFootEvent('EMAILADDR', email, self.__name__, event)
+                                self.notifyListeners(evt)
+
+                    domain_cert_summary = domain_item.get("cert_summary")
+                    if domain_cert_summary:
+                        domain_cert_summary_subject = domain_cert_summary.get("subject")
+                        if domain_cert_summary_subject:
+                            if domain_cert_summary_subject.get("organization"):
+                                evt = SpiderFootEvent('SSL_CERTIFICATE_ISSUED',
+                                                      domain_cert_summary_subject["organization"], self.__name__,
+                                                      event)
+                                self.notifyListeners(evt)
+
+                        domain_cert_summary_issuer = domain_cert_summary.get("issuer")
+                        if domain_cert_summary_issuer:
+                            if domain_cert_summary_issuer.get("organization"):
+                                evt = SpiderFootEvent('SSL_CERTIFICATE_ISSUER',
+                                                      domain_cert_summary_issuer["organization"], self.__name__,
+                                                      event)
+                                self.notifyListeners(evt)
+
+                    domain_trackers = domain_item.get("trackers")
+                    if domain_trackers:
+                        if domain_trackers.get("google_analytics_key"):
+                            evt = SpiderFootEvent('WEB_ANALYTICS_ID', domain_trackers["google_analytics_key"],
+                                                  self.__name__, event)
```

Comment:
SpiderFoot uses a specific format for `WEB_ANALYTICS_ID`.

```suggestion
                            evt = SpiderFootEvent("WEB_ANALYTICS_ID", f"Google Analytics: {domain_trackers.get("google_analytics_key")}", self.__name__, event)
```

Reply:
Thanks!